proposals/description/tip_115.ts

```diff
@@ -51,9 +51,60 @@ const tip_115: TemplatedProposalDescription = {
       method: 'acceptBeneficiary()',
       arguments: (addresses) => [],
       description: 'Accept beneficiary of rari infra tribe timelock'
+    },
+
+    //// Trigger new LBP of ETH
+    {
+      target: 'pcvGuardianNew',
+      values: '0',
+      method: 'withdrawToSafeAddress(address,address,uint256,bool,bool)',
+      arguments: (addresses) => [
+        addresses.daiFixedPricePSM,
+        addresses.ethToDaiLBPSwapper,
+        '1000000000000000000000000',
+        false,
+        false
+      ],
+      description: 'Withdraw 1M DAI to LBP Swapper'
+    },
+    {
+      target: 'pcvGuardianNew',
+      values: '0',
+      method: 'withdrawToSafeAddress(address,address,uint256,bool,bool)',
+      arguments: (addresses) => [
+        addresses.ethPSM,
+        addresses.ethToDaiLBPSwapper,
+        '10000000000000000000000',
+        false,
+        false
+      ],
+      description: 'Withdraw 10k ETH to LBP Swapper'
```

Comment:
Weights are 95 / 5 . You're sending $1m DAI and ~$10.5m ETH. So overfunding by $500k, should be fine